examples/basic-nextjs/src/app/not-found.tsx

```diff
@@ -0,0 +1,5 @@
+import NotFound from 'src/NotFound';
+
+export default function GlobalNotFound() {
+  return <NotFound />;
```

Comment:
Please, reference our current implementation, it should support custom error pages